amida-tech/greyscale
# Task/plugin auth service

#### What's this PR do?
Connects Indaba backend to the auth service
#### Related JIRA tickets:
#### How should this be manually tested?
- Run the auth service: https://github.com/amida-tech/amida-auth-microservice
- Run the greyscale backend
- Make sure the same user exist in both DB's. 
    - Create a user using post man on the auth service: http://localhost:4000/api/user
    - Create a user on indaba backend by editing the config file: 
      ```testEntities: { 
               admin: { ... }
           }```
        and then run: node_modules/.bin/mocha test/user.integration.js --bail -t 10000
- Log in on the auth service and retrieve token: http://localhost:4000/api/auth/login
- Use token to access any protected route on greyscale. For example: http://localhost:3005/testorg/v0.2/users/self
#### Any background context you want to provide?
Please note the user integration test currently doesn't pass because we need to modify it to work with the auth service. 
#### Screenshots (if appropriate):
